src/main/java/no/sikt/nva/validators/DublinCoreValidator.java

```diff
@@ -192,15 +229,13 @@ private static Optional<ErrorDetails> getInvalidTypes(DublinCore dublinCore) {
         }
     }
 
-    private static Optional<ErrorDetails> getIssnErrors(DublinCore dublinCore, BrageLocation brageLocation) {
-        if (hasIssn(dublinCore)) {
-            var issn = DublinCoreScraper.extractIssn(dublinCore, brageLocation);
-            ISSNValidator validator = new ISSNValidator();
-            if (!validator.isValid(issn)) {
-                return Optional.of(new ErrorDetails(Error.INVALID_ISSN, List.of(issn)));
-            }
-        }
-        return Optional.empty();
+    private static boolean isJournalArticle(DublinCore dublinCore) {
+        return DublinCoreScraper.extractType(dublinCore).get(0).equals(BrageType.JOURNAL_ARTICLE.getValue());
+    }
```

Comment:
Har vi ikke lagd en egen klasse for å hente å konvertere Brage typer til NVA typer? Kan noe gjenbrukes derfra?

Reply:
Vi har en metode ja, men den sjekker på dc value og ikke dublinCore . Ved å bruke extractJournal fra DublinCoreParser kan jeg ende opp med "empty" verdi, da vi returnerer new DcValue dersom fil ikke har Journal feltet. Så jeg gjenbruker  extractJournal, men siden returnerte verdien er streng, og ikke DcValue, må jeg sjekke om den strengen faktisk er Journal. hvis det gir mening :)